Lib/test/test_asyncio/test_sslproto.py

```diff
@@ -120,7 +121,19 @@ def test_close_during_handshake(self):
         test_utils.run_briefly(self.loop)
 
         ssl_proto._app_transport.close()
-        self.assertTrue(transport.abort.called)
+        self.assertTrue(transport._force_close.called)
+
+    def test_close_during_ssl_over_ssl(self):
+        # gh-113214: passing issues from the inner wrapped SSL protocol to the
+        # shim transport provided by the outer SSL protocol should not raise
+        # attribute errors
+        outer = self.ssl_protocol(proto=self.ssl_protocol())
+        self.connection_made(outer)
+        # Closing the outer app transport should not raise issues
```

Comment:
Here and in the previous comment you refer to "issues". Maybe you meant "exceptions" both cases?